node/src/accountant/scanners/payable_payments_agent_web3.rs

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+
+use crate::accountant::scanners::payable_payments_agent_abstract_layer::PayablePaymentsAgent;
+use crate::db_config::persistent_configuration::{PersistentConfigError, PersistentConfiguration};
+use crate::sub_lib::blockchain_bridge::ConsumingWalletBalances;
+use web3::types::U256;
+
+#[derive(Debug, Clone)]
+pub struct PayablePaymentsAgentWeb3 {
+    gas_limit_const_part: u64,
+    upmost_added_gas_margin: u64,
+    consuming_wallet_balance_opt: Option<ConsumingWalletBalances>,
+    pending_transaction_id_opt: Option<U256>,
+    gwei_per_computed_unit_opt: Option<u64>,
+}
+
+impl PayablePaymentsAgent for PayablePaymentsAgentWeb3 {
+    fn conclude_required_fee_per_computed_unit(
+        &mut self,
+        consultant: &dyn PersistentConfiguration,
```

Comment:
As long as this is typed with `PersistentConfiguration`, rather with some trait that `PersistentConfiguration` satisfies, I'd like to see this named `persistent_config` the way it is in other places in the codebase for consistency.